Raven.Client.Lightweight/Convention.cs

```diff
@@ -29,9 +29,9 @@ public FailoverBehavior FailoverBehaviorWithoutFlags
         }
 
         /// <summary>
-        /// Enable multipule async operations
+        /// Enable multiple async operations
         /// </summary>
-        public bool AllowMultipuleAsyncOperations { get; set; }
+        public bool AllowMultipleAsyncOperations { get; set; }
```

Comment:
This is a breaking change in the API.
We can't apply it in 3.x - Can you send the same PR for 4.0?